tokens/README.md

```diff
@@ -125,3 +125,113 @@ recipient address. That SOL can be used to pay transaction fees on staking
 operations such as delegating stake. The rest of the allocation is put in
 a stake account. The new stake account address is output in the transaction
 log.
+
+## Distribute SPL tokens
+
+Distributing SPL Tokens works very similarly to distributing SOL, but requires
+the `--owner` parameter to sign transactions. Each recipient account must be an
+initialized SPL Token Account that shares the same mint as the sender in order
+for an allocation to succeed.
```

Comment:
> Each recipient account must be an
initialized SPL Token Account that shares the same mint as the sender in order
for an allocation to succeed.

I think I'd like to got all-on on DTA here.  Each recipient account must be a system account and the recipient's token account will be derived with DTA.  If the DTA does not exist, it will be funded as a part of token distribution (perhaps a command-line option eventually, for now I think it's ok to hard code always funding)

Reply:
Just to confirm, DTA will work for your Feature Proposal use-case? (My understanding was that you wanted to transfer directly to know token accounts.)

Reply:
One other question. For the source account, I think we can either:
(1) pass in an SPL token account address and owner, as it does in this draft, or
(2) pass in a system account sender_keypair and send from its DTA; I believe this requires passing in the mint.
I prefer #2, but I'd like to confirm that passing in the mint will not be a hardship?

Reply:
> Just to confirm, DTA will work for your Feature Proposal use-case? (My understanding was that you wanted to transfer directly to known token accounts.)

Yes, even preferred.  I want to provide a list of validator identity keypairs, and have the tokens distributed to their DTAs.  


> One other question. For the source account, I think we can either:
> (1) pass in an SPL token account address and owner, as it does in this draft, or
> (2) pass in a system account sender_keypair and send from its DTA; I believe this requires passing in the mint.
> I prefer (2), but I'd like to confirm that passing in the mint will not be a hardship?

(1) wfm.  https://github.com/solana-labs/solana-program-library/pull/824 creates a token account (and mint) at program-derived addresses.   There actually is no system account for the source in this case.